rfc6962-bis.xml

```diff
@@ -1308,6 +1308,14 @@ but it is expected there will be a variety.
         <t>
           If the TLS client holds an STH that predates the SCT, it MAY, in the process of auditing, request a new STH from the log (<xref target="get-sth"/>), then verify it by requesting a consistency proof (<xref target="get-sth-consistency"/>). Note that if the TLS client uses <spanx style="verb">get-all-by-hash</spanx>, then it will already have the new STH.
         </t>
+        <section title="TLS Feature Extension">
+          <t>
+            If any certificate in a chain includes the <xref target="tls_transinfo_extension">transparency_info</xref> TLS extension identifier in the <xref target="RFC7633">TLS Feature</xref> certificate extension, then CT compliance (using any of the mechanisms from <xref target="tls_servers"/>) is required.
```

Comment:
This seems a little less clear than the original language - is compliance required for every cert in the chain? Just the EE cert? All certs from the marked cert downward? (I prefer that answer, btw).